core/import-csv.c

```diff
@@ -104,8 +104,8 @@ static char *parse_dan_new_line(char *buf, const char *NL)
 
 static int try_to_xslt_open_csv(const char *filename, struct memblock *mem, const char *tag);
 static int parse_dan_format(const char *filename, char **params, int pnr, struct dive_table *table,
-			    struct trip_table *trips, filter_preset_table_t *filter_presets,
-			    struct dive_site_table *sites)
+			    struct trip_table *trips, struct dive_site_table *sites,
+			    filter_preset_table_t *filter_presets)
```

Comment:
based on your own include file, for C code `filter_preset_table_t *` is the same as `void *` thanks to https://github.com/subsurface/subsurface/blob/2e5913d2ba6c5b5c384121692045be52836ac6fa/core/filterpreset.h#L31
And since `void *` is compatible with any pointer in C...

Reply:
Argh. That's stupid. We have to change that.